server/homebrew.api.js

```diff
@@ -54,6 +54,13 @@ const api = {
 				});
 			stub = stub?.toObject();
 
+			if(stub.lock?.state) {
+				// State 1 : Locked for everything
+				// State 2 : Edit only
+				if(stub.lock.state == 1 || (stub.lock.state == 2 && accessType != 'edit'))
+					throw { HBErrorCode: '100', code: stub.lock.code, message: stub.lock.message, brewId: accessType === 'edit' ? stub.editId : stub.shareId, brewTitle: stub.title };
```

Comment:
What is `lock.code` meant to be?

Reply:
`lock.code` contains the http response code to be sent back .. e.g. `451` (Unavailable for Legal Reasons) or `403` (Forbidden).

Reply:
Ah. Of course.